src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
+@import '../styles/variables';
+@import '../styles/card';
+@import '../styles/stars';
```

Comment:
Ensure that the imported SCSS files (`variables`, `card`, and `stars`) exist and contain the necessary variables and styles. This will ensure that all styles and variables are correctly applied.